### ISSUE TITLE
UI Polish: No Source Selected

### ISSUE THREAD

**Issue by ninavizz** [freedomofpress/securedrop-client#847]
## Problem
Existing implementation is too high/left on the screen. "Blank State" screens need to have all text centered, and support visual elements included so that users don't confuse text as potentially clickable. 

## Solution
See Zeplin for placement of lines and centering of text on screen: https://zpl.io/Vqv0vwJ
- Center text
- Implement line-spacing
- Add liney-lines
- Text possibly too dark... wd be good to review (I can't de-select _Sources_ right now, so cannot check that).

**Comment by ninavizz** [comment]
The TL;DR on how it needs to be different, I tried to illustrate with the pink annotation lines on this mockup...
- [ ] Include horizontal rules (shown in Zeplin mockup, above) and be mindful to keep them fixed to the same `223px` width, regardless of screen resizing.
- [ ] Content block should not resize when window resizes—just its positioning (but ideally, not by much)
- [ ] Even padding on both sides of the content block, and on the top/bottom of the content block.
- [ ] Asymmetric padding between above and below the text, and the horizontal rules.

![image](https://user-images.githubusercontent.com/8262612/75929957-7735bc80-5e26-11ea-99fe-b4f3700c9c14.png)
